### ISSUE TITLE
Question about the self-training process

### ISSUE THREAD

**Issue by LydJason** [RozDavid/UnScene3D#12]
Thank you very much for your outstanding work! I have a question regarding the self-training part. I don't quite understand why self-training can significantly improve performance. It uses pseudo masks that contain some errors, such as over- or under-split masks as 'ground truth' supervision, but ultimately it can obtain better instance masks than 'ground truth'. Does it mean some of the errors were corrected through the training process? This seems counterintuitive. Is this related to model underfitting/overfitting? I'm very curious about this, and I think there might be a misunderstanding on my part. Could you please explain this to me? Thank you!
